.github/workflows/go.yaml

```diff
@@ -0,0 +1,45 @@
+name: Go
+on: [push]
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Setup Go
+      uses: actions/setup-go@v2-beta
+      with:
+        go-version: '^1.14'
+        
+    - name: Install goimports
+      run: go get golang.org/x/tools/cmd/goimports
+
+    - name: Check code
+      uses: actions/checkout@v2
+    
+    - name: goimports
+      run: test -z "$($(go env GOPATH)/bin/goimports -l .)"
```

Comment:
Does this recurse into submodules? The existing CI implementation recently added lots of extra bash logic for this.

Reply:
Yes, I don't think goimports respects modules.

Reply:
Confirmed. It also finds fault with generated `*pb.go` files, so we may need exemption logic.

Reply:
Perhaps:

```
find . -name go.mod -execdir $(go env GOPATH)/bin/goimports -l . \;
```

I confirmed this finds a formatting issue in a submodule. It also doesn't say anything about `*pb.go` files.

Reply:
I don't think we have to do this. goimports does not seem to respect modules from the test I ran.

Reply:
Ah. Even better!